special_edition/asteroid_city.yaml

```diff
@@ -0,0 +1,33 @@
+accent: '#D83E1C'
+background: '#c4f4f1'
+details: darker
+foreground: '#22211a'
+
+############################################################### SEE BELOW
+background_image:
+  # the path is relative to ~/.warp/themes/
+  # the full path to the picture is: ~/.warp/themes/warp.jpg
+  path: special_edition/asteroid_city_wallpaper.jpg
+  # the opacity value is required and can range from 0-100
```

Comment:
nit: might be best to link the source of the image here, just so we don't run into any copyright issues